scripts/pylib/twister/twisterlib.py

```diff
@@ -2912,6 +2923,12 @@ def run(self):
             if instance.handler.type_str == "device":
                 instance.handler.suite = self.suite
 
+            if(self.seed is not None and instance.platform.name.startswith("native_posix")):
```

Comment:
instead of hardcoding this native_posix, I would add a new attribute (seed: true, or whatever)  to the platform yaml, i.e. `boards/posix/native_posix/native_posix.yaml`  and just check for that here.

Reply:
@nashif - What I'm trying to achieve here is to reduce the parsing of defined configs here to only POSIX to reduce any overhead for other platforms as they will never have `FAKE_ENTROPY_NATIVE_POSIX` defined.  At this point in the script the defined configs are not populated. Adding a new attribute to platform yaml to populate defined configs doesn't feel right to me.  I can use `instance.platform.arch == "posix"` instead which will include the `native_posix_64` platform variant as this is part of the platform yaml.  What are your thoughts?

Reply:
actually the data is available at this point,  I am talking about the yaml file for the platform which is one of the first things to be parsed and can be used for this type of checks.